optuna/pruners/_wilcoxon.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import warnings
+
+import numpy as np
+
+import optuna
+from optuna._experimental import experimental_class
+from optuna.pruners import BasePruner
+from optuna.study._study_direction import StudyDirection
+from optuna.trial import FrozenTrial
+
+
+@experimental_class("3.6.0")
+class WilcoxonPruner(BasePruner):
+    """Pruner based on the Wilcoxon signed-rank test <https://en.wikipedia.org/wiki/Wilcoxon_signed-rank_test>`_.
```

Comment:
This link is not rendered as intended.
<img width="834" alt="image" src="https://github.com/optuna/optuna/assets/30489874/1aa198dd-5459-4406-8a2a-b3e884a013b5">